### ISSUE TITLE
'list' object has no attribute 'get_body' in ServiceBus trigger

### ISSUE THREAD

**Issue by Hazhzeng** [Azure/azure-functions-python-worker#783]
### TL;DR
Change the **cardinality** to **one** in ServiceBus binding **function.json**.

### Background
Recently, we upgraded the function host version to 3.0.15065 for v3. We also upgraded the Python worker from 1.1.7 to 1.1.8, and the change has been deployed to production on November 16, 2020.

In Python worker release [1.1.8](https://github.com/Azure/azure-functions-python-worker/releases/tag/1.1.8), we fix the function.json **"cardinality"="many"** handling for ServiceBus triggers.

When setting **"cardinality"="many"**, the Python handler signature should accept a list of messages instead of a single one.
```python
from typing import List
import azure.functions as func

def main(msgs: List[func.ServiceBusMessage]):
```

### Impacts on Python Functions Applications
Some ServiceBus triggers may have **"cardinality"="many"** setting in their **function.json** file, and if the ServiceBus trigger only handles a single message, this can cause your app to throw a no attribute exception.
```python
import azure.functions as func

def main(msg: func.ServiceBusMessage):
```
Since the msg parameter now has a proper list of messages, instead of a single one.

### How To Mitigate
In ServiceBus triggers **function.json**, change the **cardinality** to **one**.

Alternatively, you can keep **cardinality=many** and change the ServiceBus trigger to process multiple messages in one go. Remember to change the **name** in **function.json** into **msgs**. 
from
```python
def main(msg: func.ServiceBusMessage):
    msg.get_body()
```
to
```python
def main(msgs: List[func.ServiceBusMessage]):
    for msg in msgs:
        msg.get_body()
```

Sorry for the inconvenience, if you have more questions related to this issue, please let me know and comment below.

**Comment by Hazhzeng** [comment]
If you're facing similar issue in EventHub, please check this announcement.
https://github.com/Azure/azure-functions-python-worker/issues/699
